packages/ui/src/helpers/index.ts

```diff
@@ -1,4 +1,3 @@
 export * from './authenticator';
 export * from './accountSettings';
-export * from './storage';
```

Comment:
Is this the only helper which was being exported?

Reply:
Yes this was the only only, I cleaned up the others when removing fileuploader.